src/screens/ProfileScreen/index.js

```diff
@@ -619,7 +619,7 @@ const ProfileScreen = ({route}) => {
     return reloadFetchAnonymousPost();
   };
   return (
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <SafeAreaView style={styles.container} forceInset={{top: 'always'}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The root container has been changed from `SafeAreaProvider` to `SafeAreaView`. This is a good change as `SafeAreaView` is more suitable for individual screens while `SafeAreaProvider` should be used at the root level of your app. However, please ensure that the `forceInset={{top: 'always'}}` prop is compatible with `SafeAreaView`.
```diff
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <SafeAreaView style={styles.container} forceInset={{top: 'always'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->